fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -2367,4 +2403,208 @@ public void replayModifyTableAddOrDropColumns(TableAddOrDropColumnsInfo info) th
             olapTable.writeUnlock();
         }
     }
+
+    // the invoker should keep table's write lock
+    public void modifyTableAddOrDropInvertedIndices(Database db, OlapTable olapTable,
```

Comment:
modifyTableAddOrDropInvertedIndices is almost the same as modifyTableAddOrDropColumns, can we try to merge  them to reuse code as much as possible.

Reply:
yes, the same code will be encapsulated into functions